recipes/plio/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "plio" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a189e7ee6973a209aee81ca412144599fdf7d2aefbb03c1d93c37212be053a7b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+    - pvl
+    - protobuf
+    - gdal
+    - icu
+    - h5py
+    - pandas
+    - sqlalchemy
+    - pyyaml
+    - affine
+    - networkx
+    - scipy
+    - certifi
+    - pyproj
+    - jinja2
+  run:
```

Comment:
You should list only direct dependencies of the package here. In other words, unless you are directly using protobuf or networkx, you can lose those deps. Listing deps of deps is bad practice.

Reply:
I went through the source and checked what was directly imported. Most of these were to handle different data representations (protobuf for a specification file, hdf5 for a h5py wrapper, network for adjacency information in a data set, etc.). I was though able to refactor out a section on the `plio` side and get the host down to just python and setup tools.